your-rents-services/src/main/java/com/yourrents/services/dto/PropertyDTO.java

```diff
@@ -0,0 +1,33 @@
+package com.yourrents.services.dto;
+
+/*-
+ * #%L
+ * Your Rents Services
+ * %%
+ * Copyright (C) 2019 - 2021 Your Rents Team
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+@Builder
+@Getter
+@ToString
+public class PropertyDTO {
```

Comment:
Deve proprio chiamarsi PropertyDTO? E' già nel package ../dto/, non potrebbe chiamarsi semplicemente Property?

(lo sai che detesto i dto...almeno che si nascondano :) )

Reply:
Ok cambio. Se vuoi li togliamo i DTO, però boh mi sembra brutto blindare il FE (REST consumer) al DB.

Reply:
No no, in questo caso sono necessari, il model jooq è troppo particolare e specifico per il db.